packages/dropdown/src/tokens.scss

```diff
@@ -0,0 +1,4 @@
+$option-height: 36px;
```

Comment:
Setting a min height for individual options should let options with long text wrap to the next line. I had https://github.com/Autodesk/hig/issues/774 in mind.